etcd3/client.py

```diff
@@ -99,13 +128,14 @@ def __call__(self, context, callback):
 
 
 class Etcd3Client(object):
-    def __init__(self, host='localhost', port=2379,
+    def __init__(self, host='localhost', port=2379, endpoints=None,
```

Comment:
The type of `endpoints` here is not documented by any docstring and looking at the code it seems to be a dict? I'd expect it to be a list. That needs clarification as to why it'd need to be a dict.

Reply:
I was undecided - probably a list of endpoints is the most user-friendly thing to do, yes.

Reply:
Yes, I was still unsure what format to ask as an input; probably a list feels more natural.